wgsl/index.bs

```diff
@@ -3067,35 +3067,55 @@ Issue: Which index is used when it's out of bounds?
   <tr><td>*e1* : *T*<br> *e2* : *T*<br> *T* is *FloatVec*<td>`e1 % e2` : *T*<td>Component-wise floating point modulus (OpFMod)
 </table>
 
+
 <table class='data'>
-  <caption>Binary arithmetic expressions with mixed scalar, vector, and matrix operands</caption>
+  <caption>Binary arithmetic expressions</caption>
   <thead>
-    <tr><th>Precondition<th>Conclusion<th>Notes
+    <tr><th>Category<th>Preconditions<th>Conclusions
   </thead>
-  <tr><td>*e1* : f32<br>
-          *e2* : *T*<br>
-          *T* is *FloatVec*
-      <td>`e1 * e2` : *T*<br>
-          `e2 * e1` : *T*
-      <td>Multiplication of a vector and a scalar (OpVectorTimesScalar)
-  <tr><td>*e1* : f32<br>
-          *e2* : *T*<br>
-          *T* is mat*N*x*M*&lt;f32&gt;
-      <td>`e1 * e2` : *T*<br>
-          `e2 * e1` : *T*
-      <td>Multiplication of a matrix and a scalar (OpMatrixTimesScalar)
-  <tr><td>*e1* : vec*M*&lt;f32&gt;<br>
-          *e2* : mat*N*x*M*&lt;f32&gt;
-      <td>`e1 * e2` : vec*N*&lt;f32&gt;<br>
-      <td>Vector times matrix (OpVectorTimesMatrix)
-  <tr><td>*e1* : mat*N*x*M*&lt;f32&gt;<br>
-          *e2* : vec*N*&lt;f32&gt;
-      <td>`e1 * e2` : vec*M*&lt;f32&gt;<br>
-      <td>Matrix times vector (OpMatrixTimesVector)
-  <tr><td>*e1* : mat*K*x*N*&lt;f32&gt;<br>
-          *e2* : mat*M*x*K*&lt;f32&gt;<br>
-      <td>`e1 * e2` : mat*M*x*N*&lt;f32&gt;<br>
-      <td>Matrix times matrix (OpMatrixTimesMatrix)
+  <tr>
+    <td>Vector OP Scalar "splats"
+    <td>V: vecN<S>; S: f32 or i32 or u32
```

Comment:
This isn't rendering properly for me. The &lt;S&gt; disappears and instead the S  section gets strikethrough.
```suggestion
    <td>V: vecN&lt;S&gt;<br> S: f32 or i32 or u32
```

Reply:
This issue appears for all rows except matrix arithmetic.